### DISCUSSION TITLE
Make unseen dates more intuitive

### DISCUSSION THREAD

**Discussion by tyggero** [lukevella/rallly#165] [Ideas]
Hi, thanks for this tool - it is perfectly simple and does just what it needs to. The new version looks awesome too.

I just want to report that some of the people I sent a poll to hadn't noticed the (+ X more...) button and filled only the first screen (it was 6 days i think) options, missing the other 10 I sent.

And I agree the current way of displaying more options is not very intuitive. Maybe it would help to have some visual edge of the window, or maybe have the arrow in every row? Just something that would make it clearer you aren't seeing all there is. 
Personally, I would love if the table supported native browser side-scrolling with shift+scrolling wheel (or swiping two fingers to the side on a touchpad).

**Comment by lukevella** [comment]
Hi @tyggero, thanks for the feedback. It's been mentioned before (See: #143) so you're not alone. I will try to address this an update very soon.

Just want to add: I find it works really well when using the keyboard by Tabbing across the checkboxes but I understand that not everyone uses it in this way.

**Reply by RuiCunhaM** [reply]
Hello! 
As a follow up to this, without creating a new discussion, I was just wondering if there is something currently in the works to address this issue. I found myself in the same situation where with multiple options it is a bit hard to notice them. 

In situations where there are many options per day, multiple days of the week can sometimes be difficult to visualize them all. I wonder if a weekly view similar to what Doodle does would be achievable? 

Anyways thanks for this great tool! :smile:

**Reply by lukevella** [reply]
Thanks appreciate you not opening a new discussion for this. I'm still working towards this but my understanding is that it's not a common enough occurrence that would justify pushing this feature to the front of the queue. I'm working through the backlog as fast as I can but it's probably more than a month away from being made available.
